### PR TITLE
explicit guide to attributes descriptions

### PR DESCRIPTION
**Pull request by alegmal**
### Summary

For reference there is a complete spec for KongIngress but it's not that clear at that point of reading where to find more information about the described attributes (slots, hash_on, hash_fallback, etc`...) even though links are provided few lines before that. 

### Reason

Explicitly providing source of information just as full spec is shown should reduce confusion.

additional prove that its not clear:
https://discuss.konghq.com/t/what-is-slots-in-kong-upstreams/665

### CODE REVIEW

File: app/kubernetes-ingress-controller/2.1.x/references/custom-resources.md [hunk 1]
```diff
@@ -249,7 +249,7 @@ Please follow the
 [Using the KongIngress resource](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-kongingress-resource)
 guide for details on how to use this resource.
 
-For reference, the following is a complete spec for KongIngress:
+For reference, the following is a complete spec for KongIngress (for properties information see [Upstream][kong-upstream], [Service][kong-service] and [Route][kong-route] entities):
```
Comment (falondarville) [inline app/kubernetes-ingress-controller/2.1.x/references/custom-resources.md:252]:
```suggestion
For reference, the following is a complete spec for KongIngress (for properties information see [Upstream][/gateway/latest/admin-api/#upstream-object], [Service][/gateway/latest/admin-api/#service-object] and [Route][/gateway/latest/admin-api/#route-object] entities):
```

Thank you for this clarification. I've updated the links as we're moving away from link variables. Would you accept this change?

Comment (alegmal) [reply app/kubernetes-ingress-controller/2.1.x/references/custom-resources.md]: Sure I'll!

**Review by lena-larionova — APPROVED** [review]

**Comment by alegmal** [comment]
@lena-larionova I think something is not alright https://docs.konghq.com/kubernetes-ingress-controller/2.1.x/references/custom-resources/#kongingress

**Comment by lena-larionova** [comment]
@alegmal thanks for checking it! Yes, the formatting was broken. I'm fixing it here now: https://github.com/Kong/docs.konghq.com/pull/3558

**Pull request merged**